qml/pages/VideoPlayer.qml

```diff
@@ -0,0 +1,140 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import QtMultimedia 5.6
+import "../components/"
+
+Page {
+    id: videoPage
+
+    // The effective value will be restricted by ApplicationWindow.allowedOrientations
+    allowedOrientations: Orientation.All
+
+    backNavigation: false
+
+    property string videoFile
+    property double controlsOpacity: 1.0
+
+    Rectangle {
+        width: parent.width
+        height: parent.height
+        color: "black"
+
+        MediaPlayer {
+            id: player
+
+            source: "file://" + videoFile
+            autoPlay: true
+
+            onPositionChanged: {
```

Comment:
I think you should be able to remove this and set "value: player.position" in Slider.

Reply:
This really seems required, otherwise slider position won't get updated after using it...